confidence/confidence.go

```diff
@@ -0,0 +1,198 @@
+package confidence
+
+import (
+	"context"
+	"fmt"
+	"time"
+	"net/http"
+	"reflect"
+	"strings"
+)
+
+type FlagResolver interface {
+	resolveFlag(ctx context.Context, flag string, defaultValue interface{},
+		evalCtx map[string]interface{}, expectedKind reflect.Kind) InterfaceResolutionDetail
+}
+
+type ContextProvider interface {
+	GetContext() map[string]interface{}
+}
+
+var (
+	SDK_ID      = "SDK_ID_GO_PROVIDER"
```

Comment:
Check this ID. Is there another one to set for Confidence SDK?

Reply:
i think it's _Confidence instead of provider.